README.md

```diff
@@ -1 +1,43 @@
-# lotto_game
\ No newline at end of file
+# Lotto Game
+Project divided into 3 learning paths, based on the lottery game.
+## Learning Path 1
+In this learning path the software allow the user to generate lottery bills.
+### To start the program:
+Run the file lotto_game.py.
```

Comment:
I see more information about the structure but less about how to run it, whether it requires arguments or a description of the program itself. What is the Lotto? How does this game work? 

If you add the correct docs string in the code you can avoid adding this in the README